integrated_channels/integrated_channel/course_metadata.py

```diff
@@ -51,7 +51,6 @@ def get_course_runs(user, enterprise_customer):
 
     enterprise_courses = client.get_enterprise_courses(
         enterprise_customer,
-        traverse_pagination=True
```

Comment:
removal to this flag is intentional? Perhaps we could pass this flag `traverse_pagination` to the method itself.

Reply:
Yes, we have to remove this as we are getting all course runs by default. This method is not being used anywhere else, so it is safe to remove the parameter.